modules/Assay/Parameter.yaml

```
@@ -3,33 +3,33 @@ enums:
     permissible_values:
       Cy3:
         description: Fluorescent dye used for the green channel on an array
-        meaning: http://purl.obolibrary.org/obo/FBbi_00000449
+        source: http://purl.obolibrary.org/obo/FBbi_00000449
       Cy5:
         description: Fluorescent dye used for the red channel on an array
-        meaning: http://purl.obolibrary.org/obo/FBbi_00000450
+        source: http://purl.obolibrary.org/obo/FBbi_00000450
   GenePerturbationEnum:
     permissible_values:
       knockdown:
         description: Gene knockdown refers to techniques by which the expression of one or more of an organism's genes is reduced, either through genetic modification (a change in the DNA of one of the organism's chromosomes) or by treatment with a reagent such as a short DNA or RNA oligonucleotide with a sequence complementary to either an mRNA transcript or a gene.
-        meaning: http://www.bioassayontology.org/bao#BAO_0002433
+        source: http://www.bioassayontology.org/bao#BAO_0002433
       knockout:
         description: A gene knockout is a genetic technique in which an organism is engineered to carry genes that have been made inoperative (have been 'knocked out' of the organism).
-        meaning: http://www.bioassayontology.org/bao#BAO_0002441
+        source: http://www.bioassayontology.org/bao#BAO_0002441
       non-targeting control:
         description: Use of a guide RNA that is designed to not target any gene in particular, used as a control for a targeting gRNA
       overexpression:
         description: Gene overexpression refers to genetic techniques in which an organism or cell line is engineered to express increased levels of a gene.
-        meaning: http://www.bioassayontology.org/bao#BAO_0002442
+        source: http://www.bioassayontology.org/bao#BAO_0002442
   GenePerturbationTechnologyEnum:
     permissible_values:
       CRE Recombinase:
         description: CRE Recombinase catalyses site-specific recombination between two 34 base pair loxp sites and maintains the phage genome as a monomeric unit-copy plasmid in the lysogenic state.
-        meaning: http://purl.obolibrary.org/obo/NCIT_C17285
+        source: http://purl.obolibrary.org/obo/NCIT_C17285
       CRISPR:
         description: ''
       RNAi:
         description: High throughput sample analysis of RNAi molecules for potential application in gene knockdown or gene silencing of target genes
-        meaning: http://purl.obolibrary.org/obo/ERO_0001688
+        source: http://purl.obolibrary.org/obo/ERO_0001688
```

```suggestion
        source: http://purl.obolibrary.org/obo/ERO_0001688
```

I'm confused - I thought that for pURLs, we should use `meaning` and for other sources use `source` - did I misunderstand? Or were these changed to source by accident?

---

https://github.com/nf-osi/nf-metadata-dictionary/pull/353/files#r1357085894